.github/workflows/api-audit.yml

```diff
@@ -25,6 +25,7 @@ jobs:
               run: |
                 cd api
                 python -m pip install --upgrade pip
+                python -m pip install --upgrade setuptools
```

Comment:
This seems to indicate then that the issue is in the base install of python that we set up in actions/setup-python above?

Reply:
It is - hence why we need to update it before running the security audit